lib/log.js

```diff
@@ -16,5 +16,24 @@ module.exports = (file) => {
             new transports.Console()
         ]
     })
-    return logger        
+
+    const obj = {
+        info: (...args) => {
+            args.forEach((e) => {
+                logger.info(JSON.stringify(e))
```

Comment:
If I pass in 5 arguments, I wouldn't want to log 5 times. I think its better to log 1 time as a long string.

1. Helper function that maps through `args`. If the element type is Error, then return `e.toString()`. Otherwise, return the element
2. The helper function returns the string generated from `util.format('%j', newArray)`. I use `util.format` because I think `JSON.stringify puts too many quotes, makes it hard to see.`
3. Then I will just log once: `logger.info( processArgs(args) )`